packages/core/src/editor/Block.css

```diff
@@ -14,23 +14,32 @@ BASIC STYLES
 }
 
 .bn-block-content {
+  display: flex;
   transition: font-size 0.2s;
   width: 100%;
 }
 
-.bn-block-content::before {
-  /* content: ""; */
-  transition: all 0.2s;
-  /*margin: 0px;*/
-}
-
 .bn-block-content.ProseMirror-selectednode > *,
 /* Case for node view renderers */
 .ProseMirror-selectednode > .bn-block-content > * {
   border-radius: 4px;
   outline: 4px solid rgb(100, 160, 255);
 }
 
+.bn-block-content::before {
+  transition: all 0.2s;
+  /* Workaround for selection issue on Chrome, see #1588 and also here:
+  https://discuss.prosemirror.net/t/mouse-down-selection-behaviour-different-on-chrome/8426
+  The :before element causes the selection to be set in the wrong place vs 
+  other browsers. Setting no height fixes this, while list item indicators are 
+  still displayed fine as overflow is not hidden. */
+  height: 0;
+  /* Also have to center the text vertically to ensure alignment at different 
+  line heights (which we change for `.bn-inline-content` below). */
+  display: flex;
+  align-items: center;
```

Comment:
Let's also explicitly set the overflow to show. There are many CSS resets out there so if you rely on a default behavior it may not work in some projects.